### ISSUE TITLE
Symbols showing on blue background in Debian with oh-my-fish on WSL

### ISSUE THREAD

**Issue by PurpleSci** [oh-my-fish/oh-my-fish#722]
I am using fish-shell 3.0.2 on Debian (in the Windows Subsystem for Linux) with oh-my-fish installed. When I run the shell, it shows the prompt not on the black background, but on the light blue instead. 

![symbolsonblue](https://user-images.githubusercontent.com/44437701/64877980-2a351280-d65b-11e9-82b9-9d3b81af2bda.png)

This is running in Windows Terminal with Campbell theme and Consolas font (and as for oh-my-fish, the default theme).

On the plain WSL, in fact, this is even worse. Instead of showing the back part of the "fish", it shows the question mark.

![questionmark](https://user-images.githubusercontent.com/44437701/64877996-315c2080-d65b-11e9-850f-e2575cb20255.png)

Is this supposed to be like so or is this some kind of an error? If this is an error, please tell me how to fix it.

Here's what the "doctor" shows:
```
Oh My Fish version:   6
OS type:              Linux
Fish version:         fish, version 3.0.2
Git version:          git version 2.20.1
Git core.autocrlf:    no
Your shell is ready to swim.
```

**Comment by ginwsby** [comment]
Same problem. Have you found a solution?

**Comment by harshakp06** [comment]
check your `omf theme` and install required fonts and symbols.
You can check [here](https://github.com/oh-my-fish/oh-my-fish/blob/master/docs/Themes.md) for themes.

**Comment by scorphus** [comment]
Solved in oh-my-fish/theme-default#15
